tests/acceptance/organization/roles-test.js

```diff
@@ -65,64 +66,8 @@ test(`visiting ${url} shows roles`, (assert) => {
   });
 });
 
-test(`visit ${url} and click to add a user`, (assert) => {
```

Comment:
ahha! There should be some files for `organization/invite` that can be dropped then, and a router entry to remove.